apps/api/src/utils/email_handler.py

```diff
@@ -6,6 +6,7 @@
 from services.sendgrid_handler import Template
 
 IH_SENDER = ("apply@irvinehacks.com", "IrvineHacks 2024 Applications")
+REPLY_TO_HACK_AT_UCI = ("hack@uci.edu", "Hack at UCI")
```

Comment:
Issue: The `reply_to` parameter needs to be added to the `send_email` function and the `reply_to` attribute of the Mail object in `send_email` needs to be set to the provided value.

Reply:
This will result in some type errors, but I can take care of those.

Reply:
In the meantime, can you add `reply_to: Tuple[str, str]` as an argument to all of the overloads?